src/SonarQube.Client/Models/Projects.cs

```diff
@@ -19,17 +19,37 @@
  */
 
 using System;
-using System.Diagnostics;
 using Newtonsoft.Json;
 
-namespace SonarLint.VisualStudio.Integration.Service
+namespace SonarQube.Client.Models
 {
-    [DebuggerDisplay("Name: {Name}, Key: {Key}")]
-    internal class ProjectInformation
+    public class SonarQubeProject
     {
         // Ordinal comparer should be good enough: http://docs.sonarqube.org/display/SONAR/Project+Administration#ProjectAdministration-AddingaProject
         public static readonly StringComparer KeyComparer = StringComparer.Ordinal;
 
+        public string Key { get; }
+        public string Name { get; }
+
+        public SonarQubeProject(string key, string name)
+        {
+            Key = key;
+            Name = name;
+        }
+
+        public static SonarQubeProject FromDto(ProjectDTO dto)
+        {
+            return new SonarQubeProject(dto.Key, dto.Name);
+        }
+
+        public static SonarQubeProject FromDto(ComponentDTO dto)
+        {
+            return new SonarQubeProject(dto.Key, dto.Name);
+        }
+    }
+
+    public class ProjectDTO
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename class 'ProjectDTO' to match camel case naming rules, consider using 'ProjectDto'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS101)